group/templates/group/groupList.html

```diff
@@ -1,135 +1,49 @@
 {% extends "index/base.html" %}
-{% load static %}
+{% load static %} 
+{% block title_name %}
+<title>Group List</title>
+{% endblock title_name %}
+{% load group_extras %}
 {% block body_block %}
-
-<div class="container-fluid">
-  <div class="contest">
-    <br>
+<br>
+<div class="container">
+  <div class="col-lg-offset-1 col-lg-10">
     <div class="well">
-      <!-- Create/Delete group need judge auth -->
-      {% if user.has_judge_auth %}
-      <div class="form-group">
-        <a class="btn btn-sm btn-default" style="float:right"
-           href="{% url 'group:new' %}">Create</a>
-      </div>
-      {% endif %}
       <h2>Group List</h2>
     </div>
-    <div class="filter" style="float:right">
-      <div class="btn-group" data-toggle="buttons-radio" style="float:right">
-        <button id="myGroupButton" type="button" class="btn btn-primary">My Group</button>
-        <button id="allGroupButton" type="button" class="btn btn-primary">All Group</button>
-        <script>
-          $(function(){
+    <div class="btn-group" data-toggle="buttons-radio" style="float:right">
+      <button id="myGroupButton" type="button" class="btn btn-primary">
+        My Group
+      </button>
+      <button id="allGroupButton" type="button" class="btn btn-primary">
+        All Group
+      </button>
+      <script>
+        $(function(){
+          $('#allGroupTable').show();
+          $('#myGroupTable').hide();
+          $('#myGroupButton').click(function(){
+            $('#allGroupTable').hide();
+            $('#myGroupTable').show();
+            $('#allGroupButton').removeClass("active");
+          });
+          $('#allGroupButton').click(function(){
             $('#allGroupTable').show();
             $('#myGroupTable').hide();
-            $('#myGroupButton').click(function(){
-              $('#allGroupTable').hide();
-              $('#myGroup_Table').show();
-            });
-            $('#allGroupButton').click(function(){
-              $('#allGroupTable').show();
-              $('#myGroupTable').hide();
-            });
-          })
-        </script>
-      </div>
+            $('#myGroupButton').removeClass("active");
+          });
+        })
+      </script>
```

Comment:
Wonder know whether it is possible to put this part in js file.
Separate html and js / css is better for maintanance.